manager/src/main/java/com/redhat/service/smartevents/manager/api/models/responses/BridgeResponse.java

```diff
@@ -10,6 +10,9 @@ public class BridgeResponse extends BaseManagedResourceResponse {
     @JsonProperty("endpoint")
     private String endpoint;
 
+    @JsonProperty("error_endpoint")
```

Comment:
@kostola I'd personally say put this into the `parameters` of the error handler. It's an endpoint specific to only the `endpoint` error stategy. I guess the analogy would be leaking the configuration of the Webhook or Kafka error handler on the `Bridge` object itself.

 So as a caller of the API,  I can get the endpoint of my `endpoint` error handler strategy via

```
bridgeresponse.getAction().getParameters().get("endpoint");
```

Reply:
done :+1: